docs/spec.md

```diff
@@ -140,11 +140,28 @@ which may allow us to remove tuple types from StableHLO
 
 ```ebnf
 ElementType ::= BooleanType | IntegerType | FloatType | ComplexType
+              | QuantizedType
```

Comment:
Having thought about this some more, I think it would be best to express quantization in types via `TensorType | QuantizedType` rather than `ElementType ::= ... | QuantizedType`. See https://github.com/sdasgup3/stablehlo/pull/5 for details.

Reply:
Thanks for the proposal! Incorporated.